WordPress/WordPressTest/NSManagedObject+Fixture.swift

```diff
@@ -0,0 +1,24 @@
+import CoreData
+
+extension NSManagedObject {
+
+    /// Loads the contents of any given JSON file into a new NSManagedObject instance.
+    ///
+    /// This helper method is useful for Unit Testing scenarios.
+    ///
+    /// - Parameters:
+    ///   - filename: The name of the JSON file to be loaded
+    ///   - context: The managed object context to use
+    /// - Returns: A new instance with property values of the given JSON file.
+    static func fixture(fromFile filename: String, context: NSManagedObjectContext) -> Self {
```

Comment:
Nitpick: Since "file name" is made up of two words, how do you feel about using the following camel cased name?

```suggestion
    static func fixture(fromFile fileName: String, context: NSManagedObjectContext) -> Self {
```

For what is worth, notice `fileName` is the naming used in a [method this PR touches](https://github.com/wordpress-mobile/WordPress-iOS/pull/18478/files#diff-a2ec0dda4a3592793251f4f57621e00879f8710610411932210e7d2d98194ddaR17).

Reply:
Sounds good to me. I'm not good at naming things at all, happy to following existing pattern!